### PR TITLE
Feature/SC-2630 dashboard calendar improvements

### PR DESCRIPTION
**Pull request by adrianjost**
# Checkliste

## Allgemein
- [x] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-2630

Default:
image
get's now bigger on hover and in front of the other events
image
also works on mobile by long pressing

## Mehr
Weitere Informationen zur DoD [hier im Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: controllers/dashboard.js [hunk 1]
```diff
@@ -16,176 +17,178 @@ const recurringEventsHelper = require('../helpers/recurringEvents');
...
-            hours,
-            currentTimePercentage,
-            showNewReleaseModal: newRelease,
-            currentTime: moment(currentTime).format('HH:mm'),
-        });
-    });
+
+		res.render('dashboard/dashboard', {
+			title: 'Übersicht',
+			events: events.reverse(),
```
Comment (adrianjost) [inline controllers/dashboard.js]: only controller change is this. Everything else is linting

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/schulcloud-client/schulcloud-client/pullRequest?prid=4519299&bid=15223213)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/schulcloud-client/schulcloud-client/pullRequest?prid=4519299&bid=15223213)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/schulcloud-client/schulcloud-client/pullRequest?prid=4519299&bid=15223213)

**Pull request merged**